CMakeLists.txt

```diff
@@ -13,17 +15,22 @@ include_directories(/usr/include/asio)
 add_subdirectory(proto)
 include_directories(${ProtobufIncludePath})
 
-add_subdirectory(test)
-
 add_subdirectory(utility)
 add_subdirectory(rpcserver)
 add_subdirectory(rice-namenode)
+add_subdirectory(zookeeper)
+add_subdirectory(zkwrapper)
 add_subdirectory(rice-datanode)
+add_subdirectory(test)
 add_subdirectory(native-filesystem)
 
 file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/config/hdfs-default.xml
 	DESTINATION ${CMAKE_CURRENT_BINARY_DIR}/utility/config-reader/)
+file(COPY ${CMAKE_CURRENT_SOURCE_DIR}/config/hdfs-default.xml
```

Comment:
Lines 29-30 and 35-56 are not needed. They are a relic from before the config reader existed